src/components/MeasurementOverlay/CustomMeasurementOverlay.js

```diff
@@ -9,6 +9,8 @@ function CustomMeasurementOverlay(props) {
   const renderAppropriateOverlay = type => {
     const annotationType = mapAnnotationToKey(props.annotation);
     switch (annotationType) {
+      case 'countMeasurement':
+        return <CustomCountMeasurementOverlay {...props}/>;
```

Comment:
Ah sorry I forgot to mention this component. The `CustomMeasurementOverlay` was implemented for a particular customer who would like to see some different contents in the component when a ellipse measurement annotation is selected. Eventually we found that we will just make it built-in in the UI, which is the `EllipseMeasurementOverlay` component you can find in `MeasurementOverlay.js`. That means, the `CustomMeasurementOverlay` is not being used by anyone and is not showing up by default now, and we will eventually delete this componnet.

Since we are making the `countMeasurement` built-in in WebViewer, do you think it makes more sense to move this overlay to `MeasurementOverlay`? You can take a look at how it renders `EllipseMeasurementOverlay`.

Reply:
Sounds good, I will create a separate `CountMeasurementOverlay.js` and modify some rendering logic in the `MeasurementOverlay.js`.